pre_commit_hooks/trailing_whitespace_fixer.py

```diff
@@ -50,6 +58,11 @@ def main(argv=None):  # type: (Optional[Sequence[str]]) -> int
             'default: %(default)s'
         ),
     )
+    parser.add_argument(
+        '--chars',
+        help='The set of characters to strip from the end of lines.  '
+        'Defaults to all whitespace characters.',
```

Comment:
I would write this as:

```
help=(
    '...'
    '...'
),
```

just so it's slightly easier to read with all the text lined up

Reply:
Addressed in commit a33a8f0.